window.d.ts

```diff
@@ -4,9 +4,13 @@ export { };
 
 declare global {
   interface Window {
-    popupProps?: Partial<IPopupProps>
+    popupProps?: Partial<IPopupProps>;
   }
   interface ScreenOrientation {
-    lock: (orientation: 'portrait' | 'landscape' | 'portrait-primary' | 'portrait-secondary' | 'landscape-primary' | 'landscape-secondary' | 'any') => Promise<void>
+    lock: (orientation: 'portrait' | 'landscape' | 'portrait-primary' | 'portrait-secondary' | 'landscape-primary' | 'landscape-secondary' | 'any') => Promise<void>;
+  }
+
```

Comment:
There is no empty lines between previous interfaces. I suggest to choose one approach.